apps/graphql/src/types/output/Passenger.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+
+import {
+  GraphQLObjectType,
+  GraphQLString,
+  GraphQLInt,
+  GraphQLBoolean,
+} from 'graphql';
+
+export default new GraphQLObjectType({
+  name: 'Passenger',
+  description: 'Passenger details',
+  fields: {
+    bags: {
+      type: GraphQLInt,
+    },
+    birthday: {
+      type: GraphQLString,
+    },
+    category: {
+      type: GraphQLString,
+    },
+    firstname: {
+      type: GraphQLString,
+    },
+    id: {
+      type: GraphQLInt,
+    },
+    insurance_type: {
```

Comment:
please use camelCase

Reply:
Can I add camelcase eslint rule, as snake case keep happening again and again
If yes, can I put `ItinerariesMock.js` in `.eslintignore` or I could just disable this rule for this file

Reply:
> Can I add camelcase eslint rule

Please do not do it for now, as we get most the rules from `@kiwicom/eslint-config` I will have a look at it and see if we can improve it for our needs.

Reply:
Ok sure, wont touch it! I ll just fix it to `insuranceType` then